### PR TITLE
tests: Enable "dbus_api" feature build only for dbus_api tests

### PR DESCRIPTION
**Pull request by likebreath**
In this way, our integration tests exercise the same set of build features (e.g. "kvm,mshv") being used for releases.

### CODE REVIEW

File: scripts/run_integration_tests_aarch64.sh [hunk 1]
```diff
@@ -258,6 +258,14 @@ else
     exit $RES
 fi
 
+# Run tests on dbus_api
+if [ $RES -eq 0 ]; then
+    cargo build --features "dbus_api" --all --release --target $BUILD_TARGET
```
Comment (rbradford) [inline scripts/run_integration_tests_aarch64.sh:263]: Unfortunately the placement here means that all tests after this point (acpi, live migration, etc) will be with that dbus_api binary. You probably should put this at the end of the file.

Comment (likebreath) [reply scripts/run_integration_tests_aarch64.sh:263]: Right. Good catch! Fixing it.

**Comment by likebreath** [comment]
@rbradford Very funny things happen here. I force pushed to the my associated fork (`0606/dbus_tests`) with commit 23da2ac78c65643af4108b22c66ee9dbf7891a68 but the PR does not reflect that change (remains on commit 96b1af09ec52be1a9f830c5d8b16fdbf40c0adf4). Tried multiple force-push and close/reopen the PR. Does not help at all. 

I believe open a new PR on the same branch would work, but just very wired. Any experience of similar thing before?

**Comment by likebreath** [comment]
> Very funny things happen here. I force pushed to the my associated fork (`0606/dbus_tests`) with commit [likebreath@23da2ac](https://github.com/likebreath/cloud-hypervisor/commit/23da2ac78c65643af4108b22c66ee9dbf7891a68) but the PR does not reflect that change (remains on commit [96b1af0](https://github.com/cloud-hypervisor/cloud-hypervisor/commit/96b1af09ec52be1a9f830c5d8b16fdbf40c0adf4)). Tried multiple force-push and close/reopen the PR. Does not help at all.
> 

Well. This PR now reflects the correct/latest commit. It must be a problem from Github's UI.

**Review by rbradford — APPROVED** [review]
There was a GitHub incident yesterday: https://www.githubstatus.com/incidents/1g1gkh0qpyvs

**Pull request merged**